.github/workflows/cross-version-tests.yml

```diff
@@ -169,7 +169,15 @@ jobs:
       - name: Install mlflow & test dependencies
         run: |
           python --version
-          pip install --upgrade pip wheel
+          # prophet==1.0.1 can only be installed `setup.py install` fallback but this was removed
+          # in pip 23.1. See https://github.com/pypa/pip/issues/8368 more details.
+          if [ "${{ matrix.package }}==${{ matrix.version }}" = "prophet==1.0.1" ]
+          then
+            pip install -U pip<23.1
+          else
+            pip install -U pip
+          fi
+          pip install -U wheel
```

Comment:
Build logs generated 3 days ago:

https://github.com/mlflow/mlflow/actions/runs/4700124616/jobs/8334430637#step:11:140

```
  Running setup.py install for prophet: started
  Running setup.py install for prophet: still running...
  Running setup.py install for prophet: finished with status 'done'
  DEPRECATION: prophet was installed using the legacy 'setup.py install' method, because a wheel could not be built for it. pip 23.1 will enforce this behaviour change. A possible replacement is to fix the wheel build issue reported above. Discussion can be found at https://github.com/pypa/pip/issues/8368
```